src/oira/statistics/deployment/content.py

```diff
@@ -399,6 +399,88 @@ def _raw_cards(self):
                     "graph.dimensions": ["zodb_path"],
                     "graph.metrics": ["Users per Year"],
                 },
+                "height": 8,
+                "width": 8,
+            },
+            "number_of_users_and_assessments_by_tools": {
+                "name": "Number Of Users And Assessments By Tools",
+                "display": "table",
+                "query_type": "query",
+                "dataset_query": {
+                    "type": "query",
+                    "query": {
+                        "source-table": self.table_id,
+                        "expressions": {
+                            "Users per Year": [
+                                "/",
+                                ["field-id", self.fields["num_users"]],
+                                ["+", ["field-id", self.fields["years_online"]], 1],
+                            ]
+                        },
+                        "fields": [
+                            ["field-id", self.fields["zodb_path"]],
+                            ["field-id", self.fields["num_users"]],
+                            ["field-id", self.fields["num_assessments"]],
+                        ],
+                        "order-by": [
+                            ["desc", ["field-id", self.fields["num_assessments"]]]
+                        ],
+                    },
+                    "database": self.database_id,
+                },
+                "result_metadata": [
+                    {
+                        "base_type": "type/Text",
+                        "display_name": "Zo Db Path",
+                        "name": "zodb_path",
+                        "special_type": "type/PK",
+                        "fingerprint": None,
+                    },
+                    {
+                        "base_type": "type/Integer",
+                        "display_name": "Num Users",
+                        "name": "num_users",
+                        "special_type": "type/Quantity",
+                        "fingerprint": {
+                            "global": {"distinct-count": 8, "nil%": 0.0},
+                            "type": {
+                                "type/Number": {
+                                    "min": 1.0,
+                                    "q1": 1.3542486889354093,
+                                    "q3": 3.803847577293368,
+                                    "max": 20.0,
+                                    "sd": 3.632547632014008,
+                                    "avg": 3.3333333333333335,
```

Comment:
I am a bit unsure about this stuff, probably it should be removed

Reply:
Yes, AFAIK the "fingerprint" entries are just to provide information about existing cards, but don't need to be passed when creating new ones.